src/css/base.typography.css

```diff
@@ -169,6 +169,17 @@
   text-decoration: underline;
 }
 
+.richtext .link-anchor {
+  font: inherit;
+  opacity: 0;
+  transition: opacity var(--animation-duration) var(--animation-easing);
+}
+
+.richtext *:hover > .link-anchor,
+.richtext *:focus > .link-anchor {
+  opacity: 1;
+}
+
```

Comment:
Suggestion: @Pete-Robelou What do you think about using `scroll-margin-top` on the heading elements, to give them the appropriate margin to account for the sticky header on mobile and desktop?

```csss
.richtext h2[id]
/* h2–h6 */
{
  scroll-margin-top: [header-height + buffer]
}
```

Reply:
@adamduncan Yeah good spot. I'll add this in.

Reply:
I've added the following CSS so it will apply to our regular headings as well as richtext. I've refactored a few other bits of CSS as a result of this - specifically `section` as this used an different implementation to deal with scroll offset.

I've also included `h1` within richtext as there have been a few articles from the blog content dump with a `h1`.

```
:--heading[id],
.richtext :--rt-heading[id] {
  scroll-margin-top: var(--size-scroll-offset);
}
```